docs/user/device_owner.rst

```diff
@@ -0,0 +1,45 @@
+Device Owner
+============
+
+Create Device Owner Account
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to manage Kolibri content channels, data and users, you must first create a **Device Owner** account and the **Facility**. The **Device Owner** account registration page appears when you start the server for the first time  after the installation of Kolibri and open the browser at http://127.0.0.1:8080/.
```

Comment:
Regarding the localhost link, will this differ across devices? Or is the default after `kolibri start` always localhost:8080?